README.md

```diff
@@ -1 +1,2 @@
-# uselessWeb
\ No newline at end of file
+# uselessWeb
+# nanobot was here!
```

Comment:
What i figure it out, Its that you do not need to us your terminal to clone a repository form git hub, directly for your vs with the properly extentions can be done